src/content/docs/en/guides/images.mdx

```diff
@@ -21,6 +21,10 @@ Your local images stored in `src/` can be used by all files in your project: `.a
 
 Store your images in the `public/` folder if you want to avoid any processing or to have a direct public link to them.
 
+If you need to dynamically import images from the `src/` folder, see the following recipe:
```

Comment:
```suggestion
If you need to dynamically import images from the [`srcDir`](/en/reference/configuration-reference/#srcdir) (`./src` by default), see the following recipe:
```

Reply:
Idk how rest of docs usually talk about the src dir, so it may not be needed

Reply:
For guidance, when referring to the folder specifically, we use:

the `src/` folder 

just as Jessica has done! (Nothing before `src` and when it's a folder, there should be a slash after it.)

Reply:
Feel free to resolve this then!